core/src/main/scala/com/raphtory/internals/serialisers/DynamicLoaderSerializer.scala

```diff
@@ -63,12 +63,13 @@ class DynamicLoaderSerializer(default: Serializer[DynamicLoader]) extends Serial
     // read how many classes are setup for Dynamic loading
     try {
       val n       = input.readInt()
+      val loader  = DynamicClassLoader(kryo.getClassLoader)
```

Comment:
we can revert this

Reply:
This is not needed any more at all as there is only one DynamicClassLoader so it can all be handled in the object.